app/Http/Controllers/SubscriberController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Subscriber;
+use Illuminate\Http\Request;
+
+class SubscriberController extends Controller
+{
+    public function index()
+    {
+        return view('admin.subscribers.index', [
+            'subscribers' => Subscriber::orderBy('updated_at', 'desc')->get()
+        ]);
+    }
+
+    public function new()
+    {
+        return view('admin.subscribers.new', [
+            'subscriber' => new Subscriber(),
+        ]);
+    }
+
+    public function create(Request $request)
```

Comment:
This scares me a bit. From a legal perspective, this service can't text people who haven't explicitly opted in. I'm probably being more cautious than is really necessary, but if we are going to allow new subscribers via the web UI, we might want to think about sending the subscribed message along with it so they can immediately opt out if they want. I'm still not sure if we'd be in full compliance with the law doing that though.

We can keep this around for now, but just maybe don't provide access to it through the UI until we know whether this is something we should avoid for sure.